content/applications/sales/sales/products_prices/products.rst

```diff
@@ -9,3 +9,4 @@ Manage your products
 
    products/import
    products/variants
+   product_images/product_images
```

Comment:
```suggestion
   products/product_images
```
You must move the `product_images.rst` page out of its own folder and put both the page and that folder inside the folder `products`.